frontend/plugins/vuex-persist.client.js

```diff
@@ -1,11 +1,11 @@
 import VuexPersistence from 'vuex-persist';
 import Cookies from 'js-cookie';
 
-export default ({ store, isDev }) => {
-    window.onNuxtReady(() => {
+export default ({ store }) => {
+    window.onNuxtReady(async () => {
         new VuexPersistence({
             storage: window.localStorage,
-            modules: ['preferences'],
+            modules: ['network', 'preferences'],
```

Comment:
Putting the whole `network` store into the local storage would create a lot of side effects (think about `isChangingNetwork` being forever stack being `true`). Instead, please store it inside `preferences`